DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/DelegateUserDataServiceTests.cs

```diff
@@ -259,18 +259,34 @@ public void GetDelegateUserByCandidateNumber_Returns_delegate_user()
         }
 
         [Test]
-        public void DeactivateDelegateUser_deactivates_delegate_user()
+        public void ActivateDelegateUser_deactivates_delegate_user()
         {
             using var transaction = new TransactionScope();
 
-            // given
+            // Given
             userDataService.GetDelegateUserById(1)!.Active.Should().BeTrue();
 
-            // when
+            // When
             userDataService.DeactivateDelegateUser(1);
 
-            // then
+            // Then
             userDataService.GetDelegateUserById(1)!.Active.Should().BeFalse();
         }
+
+        [Test]
+        public void ActivateDelegateUser_activates_delegate_user()

```

Comment:
Sorry, perhaps I should have been more clear. I meant that ONLY this test name should have been renamed to match the new data service method. All of the other renaming is unnecessary. So 
- the other test in this file should have remained as "DeactivateDelegateUser_deactivates_delegate_user"
- The new controller method and its route should have remained as "ReactivateDelegate"
- The new controller unit test names should have remained as "Reactivating_delegate_..."